### DISCUSSION TITLE
BFG REPO JAR

### DISCUSSION THREAD

**Discussion by Anandanumasankar** [community/community#40549] [New to GitHub]
### Select Topic Area

Question

### Body

i have a commit in my file some of the usernames must be deleted from the history some of them should not is there any way using a BFG JAR that i can do it

**Comment by jfullstackdev** [comment]
if we are referring to the same project, then go to this repo and create an issue, ask the same question

https://github.com/rtyley/bfg-repo-cleaner

**Comment by ghostinhershell** [comment]
Hi @Anandanumasankar,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [bfg-repo-cleaner](https://github.com/rtyley/bfg-repo-cleaner) repo. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
